packages/generator/src/generation/generators/table/mui-dt-generator.ts

```diff
@@ -0,0 +1,230 @@
+
+        this._imports = [...this._imports, ...this.intlFormatter.getImports()]
+
+        var uniqueImports = this.uniqueImports();
+        uniqueImports.push(TypescriptHelper.createNameSpaceImport('React', 'react'))
+        
+        return {functionDeclaration: functionalComponent, imports: uniqueImports};
+    }
+
+    getTableDefinition() : TableComponentDefinitionBase {
+        return MuiDtTableComponents;
+    }
+
+    private createStatements(): ts.Statement[] {
+      let statements = new Array<ts.Statement>()
+
+      if(this.context.useFormatter){
+        statements.push(this.intlFormatter.getImperativeHook())
+      }
+
+      let columnsIdentifier = factory.createIdentifier("columns")
+      let columnsDeclaration = this.createColumns(columnsIdentifier)
+
+      var columnsAttribute = createJsxAttribute("columns", "columns")
+      statements.push(factory.createVariableStatement(undefined, columnsDeclaration))
+
+      var rowsAttribute = createJsxAttribute("rows", this.getInputParameterIdentifier())
+
+      let returnStatement = this.createReturnStatement([columnsAttribute, rowsAttribute])
+
+      statements.push(returnStatement)
+
+      return statements;
+    }
+
+    private createReturnStatement(parameters: ts.JsxAttributeLike[]):ts.ReturnStatement {
+      var dataGridComponent = this.prepareComponent(this.getTableDefinition().table);
+
+      let wrappedTable = this.createTableWrapper(createJsxSelfClosingElement(dataGridComponent.tagName, parameters))
+
+      return factory.createReturnStatement(factory.createParenthesizedExpression(wrappedTable))
+    }
+
+    private createTableWrapper(datagrid:ts.JsxSelfClosingElement) {
+      return factory.createJsxElement(
+        factory.createJsxOpeningElement(
+          factory.createIdentifier("div"),
+          undefined,
+          factory.createJsxAttributes([factory.createJsxAttribute(
+            factory.createIdentifier("style"),
+            factory.createJsxExpression(
+              undefined,
+              factory.createObjectLiteralExpression(
+                [
+                  factory.createPropertyAssignment(
+                    factory.createIdentifier("height"),
+                    factory.createNumericLiteral("400")
+                  ),
+                  factory.createPropertyAssignment(
+                    factory.createIdentifier("width"),
+                    factory.createStringLiteral("100%")
+                  )
+                ],
+                false
+              )
+            )
+          )])
+        ),
+        [
+          factory.createJsxText(
+            "\
+            ",
+            true
+          ),
+          datagrid,
+          factory.createJsxText(
+            "\
+          ",
+            true
+          )
+        ],
+        factory.createJsxClosingElement(factory.createIdentifier("div"))
```

Comment:
Please call funtion that wraps opening and closing element https://github.com/iteria-app/lowcode/commit/745110c99c28edd6489ac545b77530ecba632b3e#diff-0c7eb128f653257f8272e20d2f7a0b6c5f1f07768644ee93f808be3383af5da8L43